css/layout.css

```diff
@@ -0,0 +1,82 @@
+.hidden {
+  display: none !important;
+}
+
+.container {
+  max-width: 576px;
+  margin: 0 auto;
+  padding: 0 1rem;
+}
+
+.row {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.column-full {
+  width: 100%;
+}
+
+.column-half {
+  flex-basis: calc(100% / 2);
+}
+
+.welcome {
+  width: 50%;
+}
+
+.nav-search {
+  width: 80px;
+}
+
+@media screen and (min-width: 769px) {
+  .container {
+    max-width: 900px;
+    margin: 0 auto;
+  }
+
+  .column-half {
+    width: 50%;
+    padding: 0.5rem;
+  }
+
+  .landing-column {
+    width: 60%;
+  }
+
+  .welcome {
+    width: 40%;
+  }
+
+  .nav-search {
+    width: 120px;
+  }
+}
+
+.otaku-row {
+  align-items: center;
+}
+
+.align-right {
+  display: flex;
+  justify-content: center;
+}
+
+.landing-row {
+  justify-content: center;
+}
+
+.search-row {
+  flex-wrap: nowrap;
+  align-items: start;
+}
+
+.search-column {
+  flex-basis: calc(100% / 2);
+}
```

Comment:
These should all go before your media queries. Media queries should always go at the bottom of the file